### ISSUE TITLE
Exemplo de uso

### ISSUE THREAD

**Issue by jonatanfelipesiqueira** [BoletoNet/BoletoNetCore#48]
Alguem tem algum exemplo de uso da biblioteca ? eu vi que tem os tests ali , mas nao consegui fazer executar,  tentei colcoar break point pra ir vendo a execucao mas n foi .

outra pergunta é, eu vi que tem como gerar uma dll da biblioteca ,  como q eu faço isso e coloco no meu projeto , pois a que esta no nuget nao possui o sicred,  é melhor gerar a biblioteca ou pegar os arquivos  e jogar pra dentro do projeto  ?

**Comment by carloscds** [comment]
@jonatanfelipesiqueira Com o projeto de testes você nao conseguiu ? https://github.com/BoletoNet/BoletoNetCore/blob/master/BoletoNetCore.Testes/BancoSicrediCarteira1Tests.cs

**Comment by jonatanfelipesiqueira** [comment]
To tentando fazer algo aqui , acho q estou no rumo,  como que eu faço pra usar o seu componente HtmlToPdf ,o exemplo vem com o Rotativa , mas nao ta funcionando aqui . porém o boleto em consegui retornar pro browser o json dele. da uma olhada ai 
![Captura de Tela 2020-03-10 às 18 25 13](https://user-images.githubusercontent.com/43968142/76360809-7e643b00-62fc-11ea-9ca2-eae2495499e9.png)

**Comment by jonatanfelipesiqueira** [comment]
to seguindo algumas coisas q encontrei no Utils

**Comment by jonatanfelipesiqueira** [comment]
> @jonatanfelipesiqueira Com o projeto de testes você nao conseguiu ? https://github.com/BoletoNet/BoletoNetCore/blob/master/BoletoNetCore.Testes/BancoSicrediCarteira1Tests.cs

Esse aqui consegui também porém faltou a geracao do pdf, ou retornar o html pro browser

**Comment by carloscds** [comment]
@jonatanfelipesiqueira O exemplo que ta no projeto esta fantando os arquivos. Você pode usar um que eu construi: https://github.com/carloscds/HtmlToPDFCore

**Comment by jonatanfelipesiqueira** [comment]
Como eu faço pra usar o seu componente ?

**Comment by jonatanfelipesiqueira** [comment]
Ok, obrigado vou testar a aviso

**Comment by jonatanfelipesiqueira** [comment]
@
![Uploading Captura de Tela 2020-03-11 às 09.41.51.png…]()

tentando usar o pacote aqui , recebi esse erro , estou usando Mac , acho q deve ser por isso

**Comment by jonatanfelipesiqueira** [comment]
![Captura de Tela 2020-03-11 às 09 43 28](https://user-images.githubusercontent.com/43968142/76418075-c4f97a00-637c-11ea-94e4-a03822f8166e.png)

**Comment by carloscds** [comment]
@jonatanfelipesiqueira Confirma se vc tem uma pasta rotativa/mac dentro do bin. Se tiver manda um print aqui do conteudo

**Comment by jonatanfelipesiqueira** [comment]
![Captura de Tela 2020-03-11 às 10 12 38](https://user-images.githubusercontent.com/43968142/76420426-d9d80c80-6380-11ea-9ba7-cf1ef11556d3.png)


esta sim, foi criado automaticamente quando eu instalei o HtmlToPDFCore , instalei direto do Nuget

**Comment by jonatanfelipesiqueira** [comment]
https://mega.nz/#!K9BVHIZb!XtV_cP4Nun_orxFt7A92czUdF6NEs1Ee6jWQzWRG2Sw

link do projeto de teste

**Comment by carloscds** [comment]
@jonatanfelipesiqueira manda o print

**Comment by jonatanfelipesiqueira** [comment]
![Captura de Tela 2020-03-11 às 10 26 48](https://user-images.githubusercontent.com/43968142/76421697-d34a9480-6382-11ea-8394-6b8894aeebaf.png)
![Captura de Tela 2020-03-11 às 10 27 11](https://user-images.githubusercontent.com/43968142/76421725-df365680-6382-11ea-8685-3fed30cb743f.png)

**Comment by jonatanfelipesiqueira** [comment]
![Captura de Tela 2020-03-11 às 10 27 38](https://user-images.githubusercontent.com/43968142/76421771-efe6cc80-6382-11ea-93e5-9162db2a86f5.png)

**Comment by jonatanfelipesiqueira** [comment]
o erro acontece na linha 

var pdf = htmlToPdf.ReturnPDF

**Comment by jonatanfelipesiqueira** [comment]
consegui fazer funcionar assim : eu renome-ei a pasta mac pra linux , o componente estava reconhecendo como linux , ain funcionou gerou o pdf na pasta, só nao exibiu na minha tela, agora deve ser algo no código . ou ainda no componente

**Comment by carloscds** [comment]
@jonatanfelipesiqueira Blz, vou ajustar o componente aqui

**Comment by jonatanfelipesiqueira** [comment]
@carloscds só me tire mais algumas duvidas, ja que voce tem mais experiencia com o componente
![Captura de Tela 2020-03-11 às 14 13 45](https://user-images.githubusercontent.com/43968142/76444306-8676b600-63a2-11ea-9954-2ec74090eac8.png)

eu vi que o componente salva o arquivo pdf na pasta do projeto , entao fiz dessa forma ai acima pra abrir o boleto na pagina, tem alguma forma melhor de se fazer  ? 

outra duvida , nao vai pesar o servidor quando comecar a ter varias geracoes de boleto ? e começar acumular no server

**Comment by jonatanfelipesiqueira** [comment]
fiz uma adaptacao aqui com o pdfmake funcionou bacana , e é bem mais leve que o Rotativa , mais tarde posto o link pra quem quiser

**Comment by carloscds** [comment]
@jonatanfelipesiqueira Não, isto é so um exemplo de uso do componente. Ele volta um array de bytes.

**Comment by jonatanfelipesiqueira** [comment]
Hum entendi , eu fiz algo aqui para nao fazer o uso do rotativa , achei muito pesado , ai retornei o html e dei um print na pagina, ja vai resolver pra mim , agora só configurar tudo aqui, muito obrigado pela ajuda. e desculpa a leiguice rsrs, sou novo no asp, programava em Delphi antes.

**Comment by jonatanfelipesiqueira** [comment]
agora vou tentar fazer o uso aqui do componente , eu fiz usando codigo da Utils, mas vou tentar com o codigo desse arquivo https://github.com/BoletoNet/BoletoNetCore/blob/master/BoletoNetCore.Testes/BancoSicrediCarteira1Tests.cs , nao sei se muda muita coisa ...

**Comment by joycecouraca** [comment]
@jonatanfelipesiqueira Conseguiu fazer funcionar ? estou precisando de ajuda se puder me dar uma luz agradeço :)
